### DISCUSSION TITLE
List와 Array문제.

### DISCUSSION THREAD

**Discussion by hjs6082** [naverz/zepeto-studio-kor#1083] [5. Q&A]
![image](https://user-images.githubusercontent.com/70927848/181168041-479f3a46-c319-4a2d-bafa-647d89c46c70.png)
이렇게 제작을 해서, 하이라키에 뜨게하고싶은데 뜨지 않습니다. 또한. Add나, push함수도 작동이 안되는데 혹시 Array나 List$1의 작동 방법이 있을까요?

**Comment by hjs6082** [comment]
하이라키가 아니라 인스펙터입니다

**Reply by whild** [reply]
엄 string[] 은 작동 안하나요??

**Reply by Sonak0930** [reply]
Array나 List는 인스펙터에 뜨지 않는거 같습니다. string[]은 아마 작동할 거에요. 그리고 string[]도 Array<String>이라서 push나 array[i]로 받아올 수 있을겁니다.

**Reply by hjs6082** [reply]
> 엄 string[] 은 작동 안하나요??

작동하네요 감사합니다!
